site/src/types/types.ts

```diff
@@ -21,6 +21,7 @@ export interface ReviewData {
   tags: string[];
   verified?: boolean;
   captchaToken?: string;
+  userVote?: number;
```

Comment:
Going off my previous comment. I see now that all reviews retrieved from the reviews route will have this attribute but reviews retrieved from the featured will not.

I think we could make this distinction clear by having another type for `FeaturedReviewData` on the frontend.

e.g. so userVote can be required here on ReviewData and featured review will be the following

```ts
export type FeaturedReviewData = Omit<ReviewData, 'userVote'>;
```

which is `ReviewData` but with the `userVote` attribute omitted.

```suggestion
  userVote: number;
```

Reply:
Done. I created the `FeaturedReviewData` type and used it in ProfessorPopup.tsx, which was the only file that was using the `/featured` endpoint.